Fakebook.Profile/Fakebook.Profile.UnitTests/APITests/ProfilePictureControllerTests.cs

```diff
@@ -84,11 +62,11 @@ public async Task UploadInvalidImageExtension(string extension)
             var file = new FormFile(stream, 0, 1000, "Data", $"dummy.{extension}")
             {
                 Headers = new HeaderDictionary(),
-                ContentType = $"image/{extension}"
+                ContentType = "application/octet-stream"
             };
 
             var httpContext = new DefaultHttpContext();
-            httpContext.Request.Headers.Add("Content-Type", "multipart/form-data");
+            httpContext.Request.Headers.Add("Content-Type", "octet-stream");
```

Comment:
no, the overall content type is supposed to be multipart/form-data for sure.

Reply:
reverted back to using multipart/form-data